### ISSUE TITLE
RPi5 Bookworm arch64 / incompatibility list

### ISSUE THREAD

**Issue by siteswapjuggler** [openframeworks/openFrameworks#7791]
So far so good but here come the first errors : 

- svgExample doesnt compile, see [svgExample_logs.txt](https://github.com/openframeworks/openFrameworks/files/13567090/svgExample_logs.txt)
- shader examples crash when the program run either from CLI with the "sudo startx [prog] -- -s off" command line or desktop environment

**Comment by fieldOfView** [comment]
It seems that the svgExample can be fixed by editing `addons/ofxSvg/addon.config.mk`:

Add 
```
linuxaarch64:
        ADDON_LIBS = libs/svgtiny/lib/linuxaarch64/libsvgtiny.a
        ADDON_LIBS += libs/libxml2/lib/linuxaarch64/libxml2.a
```
after the similar chunk for `linuxarmv7l`

**Comment by siteswapjuggler** [comment]
I'll give it a try thanks a lot for your answer :)

**Comment by siteswapjuggler** [comment]
Yep it works perfectly with this editing, we should probably search for other missing references to xml and svgtiny for linuxaarch64 in the other addons too.

Thanks for resolving that :)

**Comment by fieldOfView** [comment]
> we should probably search for other missing references to xml and svgtiny for linuxaarch64 in the other addons too

I searched for similar missing linuxaarch64 lemmas, but could not find any. So I created a PR: #7917

**Comment by fieldOfView** [comment]
Fix for the SVG addon has been merged.

**Comment by siteswapjuggler** [comment]
Hi I did test the fix for the SVG with a fresh install of the latest commit, it's ok to me. 

By th way the original issue was listing two different problems so there is still one to go. But I imagine we can close this one and start a new one with this remaining error : 

* shader examples crash when the program run either from CLI with the "sudo startx [prog] -- -s off" command line or desktop environment
